src/wmtk/simplex/cofaces_single_dimension.hpp

```diff
@@ -17,4 +18,13 @@ std::vector<Tuple> cofaces_single_dimension_tuples(
     const Mesh& mesh,
     const Simplex& my_simplex,
     PrimitiveType cofaces_type);
+
+std::vector<Simplex> cofaces_single_dimension_simplices(
```

Comment:
the naming scheme has been to just call it `cofaces_single_dimension` rpeviously - could you switch to that?